### ISSUE TITLE
pg_stat_archiver query failed: sql: Scan error on column index 6, name "date_part": converting NULL to int64 is unsupported

### ISSUE THREAD

**Issue by Brandon-Godwin** [rapidloop/pgmetrics#44]
After upgrading to AWS RDS Aurora Postgres 11.11, I started getting this error:
> pg_stat_archiver query failed: sql: Scan error on column index 6, name "date_part": converting NULL to int64 is unsupported

I was able to fix it with this patch on v1.10.5:
```patch
diff --git a/collector/collect.go b/collector/collect.go
index fe4e4f4..262ee19 100644
--- a/collector/collect.go
+++ b/collector/collect.go
@@ -611,7 +611,7 @@ func (c *collector) getWALArchiver() {
 			failed_count,
 			COALESCE(last_failed_wal, ''),
 			COALESCE(EXTRACT(EPOCH FROM last_failed_time)::bigint, 0),
-			EXTRACT(EPOCH FROM stats_reset)::bigint
+			COALESCE(EXTRACT(EPOCH FROM stats_reset)::bigint, 0)
 		  FROM pg_stat_archiver`
 	a := &c.result.WALArchiving
 	if err := c.db.QueryRowContext(ctx, q).Scan(&a.ArchivedCount, &a.LastArchivedWAL,

```

If anyone else needs to quickly get this fixed before a new release is available that fixes it, here's a Dockerfile snippet to help out building a patched version of 1.10.5.

```Dockerfile
FROM golang:1.15-buster as pgmetrics_builder

RUN git clone -b v1.10.5 https://github.com/rapidloop/pgmetrics.git
COPY pgmetrics-fix-wal-archiver-stats.patch ./
RUN cd pgmetrics \
    && git apply ../pgmetrics-fix-wal-archiver-stats.patch \
    && CGO_ENABLED=0 go build -ldflags '-s -w -X main.version=1.10.5-patched -extldflags "-static"' -trimpath ./cmd/pgmetrics


FROM your_image

COPY --from=pgmetrics_builder /go/pgmetrics/pgmetrics ./
```

**Comment by mdevan** [comment]
Thanks for reporting. Fixed in aa51b0f.

**Comment by mdevan** [comment]
This is now available in v1.12.
